constraints/cpu.md

```diff
@@ -0,0 +1,35 @@
+# CPU Constraints in Kata Containers
+
+## Default number of virtual CPUs
+
+Before starting a container, the runtime reads the option `default_vcpus` from the configuration file to determine the number of virtual CPUs (vCPUs) needed to start the virtual machine. By default, `default_vcpus` is equal to 1 for fast boot time and a small memory footprint per virtual machine. Be aware that increasing this value negatively impacts the virtual machine's boot time and memory footprint.
```

Comment:
Could you switch the word order to make this read better (and add a link to the runtime repo):

> ... the [runtime](https://github.com/kata-containers/runtime) reads the `default_vcpus` option from ...